package.json

```diff
@@ -127,29 +124,29 @@
     "webpack-node-externals": "^3.0.0"
   },
   "resolutions": {
-    "@babel/runtime": "7.26.10",
-    "sass-lint/merge": "^2.1.1",
-    "uglifyjs-webpack-plugin/serialize-javascript": "^3.1.0",
-    "@hmcts/nodejs-logging/moment": "^2.29.4",
-    "qs": "^6.5.3",
-    "json5": "^2.2.2",
-    "minimist": "^1.2.6",
-    "semver": "^7.5.2",
-    "follow-redirects": "^1.15.6",
-    "word-wrap": "^1.2.4",
+    "@babel/runtime": "^7.26.10",
+    "sass-lint/merge": ">=2.1.1",
+    "uglifyjs-webpack-plugin/serialize-javascript": ">=3.1.0",
```

Comment:
feels like all these resolutions should be hardcoded rather than >=. This reduces our risk of random stuff getting sent in out of nowhere with every change.